regression-test/suites/external_table_p0/jdbc/test_doris_jdbc_catalog.groovy

```diff
@@ -167,6 +171,12 @@ suite("test_doris_jdbc_catalog", "p0,external,doris,external_docker,external_doc
     order_qt_tb2 """ select pin_id, hll_union_agg(user_log_acct) from ${catalog_name}.${internal_db_name}.${hllTable} group by pin_id; """
     order_qt_base2 """ select * from ${catalog_name}.${internal_db_name}.${base_table} order by int_col; """
     order_qt_arr2 """ select * from ${catalog_name}.${internal_db_name}.${arr_table} order by int_col; """
+    sql """ drop table if exists internal.${internal_db_name}.ctas_base; """
+    sql """ drop table if exists internal.${internal_db_name}.ctas_arr; """
+    order_qt_ctas_base """ create table internal.${internal_db_name}.ctas_base PROPERTIES("replication_num" = "1") as select * from ${catalog_name}.${internal_db_name}.${base_table} order by int_col; """
```

Comment:
we'd better add `desc table` test for table which is created by CTAS, to ensure the type mapping is correct